scripts/update-documentation-md.ts

```diff
@@ -0,0 +1,25 @@
+import { resolve } from 'node:path';
+import { unixEOL } from '@sap-cloud-sdk/util';
+import { transformFile } from './util';
+import { exit } from 'node:process';
+
+async function updateDocumentationMd() {
+  if (!process.env.NEXT_PACKAGE_VERSION) {
+    throw new Error('NEXT_PACKAGE_VERSION is not set!');
+  }
+  await transformFile(resolve('DOCUMENTATION.md'), documentation =>
+    documentation
+      .split(unixEOL)
+      .map(line =>
+        line.startsWith('## Version:')
+          ? `## Version: ${process.env.NEXT_PACKAGE_VERSION}`
+          : line
+      )
+      .join(unixEOL)
```

Comment:
`get-changelog.ts` you decided to replace `unixEOL` with `\n` I suppose you can do the same here, since the action environment always is unix-like?